GSA/openacr-editor
# External links

Closes https://github.com/GSA/openacr/issues/297

**QA**

1. After deployment ('Deploy' action) is successful, open site https://gsa.github.io/openacr-editor/.
2. Click 'Open report'.
3. Upload a yaml file from OpenACR. Try the 'drupal-9.yaml' or any of the yaml files from https://github.com/GSA/openacr/tree/main/openacr.
4. Navigate the different pages, confirm external links have an external link icon and open in a new window/tab when clicked.
5. Click the 'View report' or 'Report' tab.
6. Confirm external links have an external link icon and open in a new window/tab when clicked.
